docs/setup/quick-start.md

```diff
@@ -0,0 +1,6 @@
+There are some ways to setup Sedna:
+- If you have already an existing Kubernetes environment, you can follow [this instruction](./install.md)
+- If you have none Kubernetes environment, you can follow [the all-in-one instruction](./all-in-one.md)
+- If you want to develop Sedna, you can try [the local-up instruction](./local-up.md)
```

Comment:
From my point of view, the third setup description is different from others.
Because the first two methods are about environment, the third is about willing.

Reply:
Change the words, you can take it a look

Reply:
+1. Good point Jensen and I am also thinking how to comment. XD

Readers will be quite confused. For example:
1) When one wants to develop Sedna, should s/he install Kubernetes? 
2) When one follow the all-in-one instruction, can s/he develop Sedna? 

The key here is to briefly explain what the difference is between all-in-one and local-up, leaving the suggestion of usage conditions as examples.